Pipfile

```diff
@@ -7,10 +7,22 @@ verify_ssl = true
 graphviz = "~=0.4"
 jupyter = "~=1.0"
 pytest = "*"
-gdcdictionary = {editable = true,git = "https://github.com/NCI-GDC/gdcdictionary.git",ref = "1.2.0"}
 
 [packages]
-gen3datamodel = {path = ".",editable = true}
+gen3datamodel = {editable = true,path = "."}
+psqlgraph = {editable = true,git = "https://git@github.com/NCI-GDC/psqlgraph.git",ref = "release/py3"}
+cdislogging = "==1.0.0"
+attrs = "==19.3.0"
+dictionaryutils = {editable = true,git = "https://github.com/uc-cdis/dictionaryutils.git",ref = "chore/py3"}
+gdcdictionary = {editable = true,git = "https://github.com/NCI-GDC/gdcdictionary.git",ref = "release/py3"}
+more-itertools = "==7.2.0"
+zipp = "==0.6.0"
+importlib-metadata = "==0.23"
+pyrsistent = "==0.15.4"
+pytz = "*"
+psycopg2-binary = "==2.8.2"
+sqlalchemy = "==1.3.3"
+rstr = "==2.2.6"
```

Comment:
how come you had to add all of these dependencies?

Reply:
pipenv kept throwing an error on the vcs depedencies so I ran `pipenv install -e <vcs dependecy> ` to autogenerate the Pipfile. It should be okay to have them here since they were in the pipfile.lock to begin with. Although, eventually once we merge all the py3 branches I'd want to move away from vcs dependencies all together so this file will likely be edited at that time to remove all these packages